modules/basics/pages/system-and-language-basics.adoc

```diff
@@ -65,6 +67,50 @@ To exit the GSQL shell, type either `exit` or  `quit` at the GSQL prompt:
 
 `GSQL> EXIT`  or `GSQL> QUIT`
 
+=== New token-based authentication method
+
+Starting with version *4.1.0*, GSQL now supports token-based authentication, allowing users to log into the GSQL shell using a token instead of a traditional username and password.
+
+*How to login using a token*
+
+To log into the GSQL shell using a token, you can use the `-t` or `--token` option, followed by the token value. The token can either be a plain-text token or a JWT token.
+[source,gsql]
+----
+gsql -t <token>
+----
+
+*Types of tokens*
+
+* *Plain-text Token:* A randomly generated string used for authentication. This is the older token format, still supported if not expired, but will eventually be deprecated.
+* *JWT Token:* A more secure and modern format for authentication. It consists of three parts: header, payload, and signature. JWT tokens are the default format for authentication starting from version 4.1.0.
+
+*Token login workflow*
+
+* *Login Process:* To log in using the `-t` option, simply provide a valid token with the `gsql -t <token>` command. If the token is valid, you will be authenticated and logged into the GSQL shell.
+* *Error Handling:* If the token is expired or invalid, the login will fail. You will be prompted with an error message stating that the token is no longer valid.
+
+*Consistency with `--secret` command*
+This token-based authentication works similarly to the existing `--secret` command, which allows you to log into GSQL using a secret instead of a password.
+
+.GSQL command example for secret login
+[source,gsql]
+----
+gsql -s <secret>
+----
+
+.GSQL command example for token login
+[source,gsql]
+----
+gsql -t <token>
+----
+
+Both methods serve as alternatives to using a username and password. The consistency between the two methods ensures a seamless user experience when switching between different authentication approaches.
+
```

Comment:
This is actually fine. There's nothing wrong with it, but it's also totally optional. It's not saying anything that users actually need to know.  I would either omit it or trim it, because users don't want to read more documentation than they need to.